src/Mod/Material/StandardMaterial/Ti-6Al-4V.FCMat

```diff
@@ -8,17 +8,21 @@
 Name = Ti-6Al-4V (Grade 5)
 Father = Metal
 KindOfMaterial = Titanium
-SourceURL = https://asm.matweb.com/search/SpecificMaterial.asp?bassnum=mtp641
 
 [Mechanical]
 Density = 4430 kg/m^3
-PoissonRatio = 0.342
+PoissonRatio = 0.330
 ShearModulus = 44 GPa
-UltimateTensileStrength = 950 MPa
-YieldStrength = 880 MPa
-YoungsModulus = 113.8 GPa
+UltimateTensileStrength = 975.5 MPa
+YieldStrength = 910 MPa
+YoungsModulus = 114 GPa
+Hardness = 35 HRC
 
 [Thermal]
-SpecificHeat = 5266.3 J/kg/K
+SpecificHeat = 553.0 J/kg/K
```

Comment:
Only a tenth now, is this correct?

Reply:
Yes, I did the conversion incorrectly originally. All other values are in this range, should be fine now.